src/blockchain/sync-augur-node-with-blockchain.ts

```diff
@@ -32,20 +32,18 @@ function getNetworkID(db: Knex, augur: Augur, callback: (err: Error|null, networ
 
 export function syncAugurNodeWithBlockchain(db: Knex,  augur: Augur, ethereumNodeEndpoints: EthereumNodeEndpoints, uploadBlockNumbers: UploadBlockNumbers, callback: ErrorCallback): void {
   augur.connect({ ethereumNode: ethereumNodeEndpoints }, (): void => {
-    console.log("Waiting for first block...");
-    startAugurListeners(db, augur, (block: Block): void => {
+    startAugurListeners(db, augur, (err: Error|null): void => {
+      if (err) return callback(err);
+      console.log("Started blockchain event listeners", augur.rpc.getCurrentBlock());
```

Comment:
We could miss blocks if we're letting both block download methods grab the currentBlock which could change between calls. However the code you're in conflicts with #93 and another refactor i am planning on making with these functions for more resilient/accurate hand-off between get-all and the blockstream and i could clean it up there

Reply:
Right, I was trying to keep it simple here b/c I know you're fixing this up properly on your branch -- please feel free to overwrite this with a proper solution :grin:

Reply:
sure thing, let's get this branch merged and i'll resolve the conflicts in mine.

In tomorrow's call, i'd like to discuss the best way to restructure augur.js, since the connect() is initializing the blockstream, but that is too early, because we don't yet have access to the rpc to start figuring out where to start/stop the sync